ui/packages/catalog-ui-search/src/main/webapp/component/metacard-overwrite/metacard-overwrite.view.js

```diff
@@ -13,116 +13,306 @@
  *
```

Comment:
Was there a reason for migrating the component to this file? I think convention is for migrated react components to go in the /webapp/react-component/presentation and /webapp/react-component/container folders. You'll also want to use typescript for the benefits of typechecking

Reply:
I think we can do this incrementally. This is a good first pass.